UnitTestApi/CMDB/ObjectTypes/ObjectTypesTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Idoit.API.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using Result = Idoit.API.Client.CMDB.ObjectTypes.Response.Result;
+using ObjectTypesRead = Idoit.API.Client.CMDB.ObjectTypes.ObjectTypes;
+using ObjectTypesCreate = Idoit.API.Client.CMDB.Object.Object;
+using System.IO;
+
+namespace UnitTestApi.CMDB.ObjectTypes
+{
+    [TestClass]
+    public class ObjectTypesTest
+    {
+        string URL;
+        string APIKEY;
+        string LANGUAGE;
+        public ObjectTypesTest()
```

Comment:
What does it test? There is no assertion here.

Reply:
This is a constructor but not a test that is used to fetch the credentials "Access data"

Reply:
Isn't there a possibility to mark this a setup for tests? For example, in JUnit (Java) you can mark test setup methods with @Before

Reply:
I think that's a very good idea.
I will research it and see if this also exists in C#

Reply:
Hi Mantas, i looked but there is no marker for the constructor

Reply:
It's not about the constructor, it's about setting up a test. Why does a test need a constructor? It's not supposed to be an object, does it?